cmd.js

```diff
@@ -97,7 +99,7 @@ function parseSubprocessCommand (args) {
       0x [flags] -- node [nodeFlags] script.js [scriptFlags]
     `)
   }
-  var pathToNodeBinary = false
+  var pathToNodeBinary = process.argv[0]
```

Comment:
I think this should *not* be `process.argv[0]`, but rather `./path/to/node` in `0x -- ./path/to/node script.js`.

Reply:
This is the fallback in case there is no extra path specified. It will then be used throughout the application.

But after looking at it again, I found that indeed that there was one case that was not covered: using it like `path/to/node cmd.js -- node script.js`. I just pushed that as well with a few other minor improvements.